react/features/base/tracks/actions.js

```diff
@@ -381,6 +381,9 @@ export function trackAdded(track) {
         track.on(
             JitsiTrackEvents.TRACK_VIDEOTYPE_CHANGED,
             type => dispatch(trackVideoTypeChanged(track, type)));
+        track.on(
+            JitsiTrackEvents.TRACK_STREAMING_STATUS_CHANGED,
+            streamingStatus => dispatch(trackStreamingStatusChanged(track, streamingStatus)));
```

Comment:
Should only video tracks listen to the `JitsiTrackEvents.TRACK_STREAMING_STATUS_CHANGED` event? Is there a use case where this might apply to audio tracks? If so, I don't think the UI supports it

Reply:
yeah I think it only makes sense for video tracks